autoload/neomake/makers/ft/python.vim

```diff
@@ -83,6 +83,11 @@ function! neomake#makers#ft#python#Flake8EntryProcess(entry) abort
     else
         let type = ''
     endif
+    let l:token = matchstr(a:entry.text, "'.*'")
+    if strlen(l:token)
+        let a:entry.length = strlen(l:token) - 2 " subtract the quotes
+    endif
```

Comment:
Well, by default the length is one column. Some makers contain information which can be used for this, e.g. to mark a non-existing function. The only maker currently setting `length` is `flake8`.

In this case, when you have a file:
```python
#!/usr/bin/python
nonexist
```

`flake8 --format=default test.py`:
```
test.py:2:1: F821 undefined name 'nonexist'
```